tools/cldr-apps/js/src/esm/cldrInfo.mjs

```diff
@@ -78,6 +82,10 @@ function insertLegacyElement(containerEl) {
   nonVueEl.className = "sidebyside-scrollable";
   nonVueEl.id = "itemInfo";
   containerEl.appendChild(nonVueEl);
+  // TODO: here, create divs inside itemInfo, each containing a part of the Info Panel
+  // which may be either a legacy div or a Vue component; never destroy these
+  // divs wholesale by calling removeAllChildNodes on itemInfo.
+  // Reference: https://unicode-org.atlassian.net/browse/CLDR-7536
```

Comment:
Right, ideally the info panel would stick around and just get updated data, never rebuilt from scratch.